<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deviceInfo</class>
 <widget class="QWidget" name="deviceInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0,0,0,0,0">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="CaptionLabel" name="protocolVersion_intro">
           <property name="text">
            <string>ADB protocol version decides the packet format between client and server. By now it has 2 versions:
01000000 used in Android versions until 8 (Oreo)
01000001 used in Android versions from 9 (Pie)</string>
           </property>
           <property name="pixelFontSize" stdset="0">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="BodyLabel" name="protocolVersion_label">
             <property name="text">
              <string>协议版本:</string>
             </property>
             <property name="pixelFontSize" stdset="0">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="BodyLabel" name="protocolVersion_value">
             <property name="text">
              <string>Body label</string>
             </property>
             <property name="pixelFontSize" stdset="0">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="CaptionLabel" name="productName_intro">
           <property name="text">
            <string>ro.product.name field in Android Build Props</string>
           </property>
           <property name="pixelFontSize" stdset="0">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="BodyLabel" name="productName_label">
             <property name="text">
              <string>产品名称:</string>
             </property>
             <property name="pixelFontSize" stdset="0">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="BodyLabel" name="productName_value">
             <property name="text">
              <string>Body label</string>
             </property>
             <property name="pixelFontSize" stdset="0">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="CaptionLabel" name="modelName_intro">
           <property name="text">
            <string>ro.product.model field in Android Build Props</string>
           </property>
           <property name="pixelFontSize" stdset="0">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="BodyLabel" name="modelName_label">
             <property name="text">
              <string>模型名称:</string>
             </property>
             <property name="pixelFontSize" stdset="0">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="BodyLabel" name="modelName_value">
             <property name="text">
              <string>Body label</string>
             </property>
             <property name="pixelFontSize" stdset="0">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="CaptionLabel" name="deviceName_intro">
           <property name="text">
            <string>ro.product.device field in Android Build Props</string>
           </property>
           <property name="pixelFontSize" stdset="0">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="BodyLabel" name="deviceName_label">
             <property name="text">
              <string>设备名称:</string>
             </property>
             <property name="pixelFontSize" stdset="0">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="BodyLabel" name="deviceName_value">
             <property name="text">
              <string>Body label</string>
             </property>
             <property name="pixelFontSize" stdset="0">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="CaptionLabel" name="deviceName_intro_2">
           <property name="text">
            <string>ro.product.device field in Android Build Props</string>
           </property>
           <property name="pixelFontSize" stdset="0">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="BodyLabel" name="resolution_label">
             <property name="text">
              <string>分辨率:</string>
             </property>
             <property name="pixelFontSize" stdset="0">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="BodyLabel" name="resolution_value">
             <property name="text">
              <string>Body label</string>
             </property>
             <property name="pixelFontSize" stdset="0">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
